assignment/scripts/loops-practice.js

```diff
@@ -12,36 +12,73 @@ for (let i=0; i<4; i++) {
 
 // 1.a. TODO: Write a for loop to console.log the numbers from 0 to 5 
 //   - Which part of the example loop do you need to change to do this?
+for (let count=0; count <= 5; count ++){
```

Comment:
Spacing is a bit odd here. The assignment operator and curly braces usually get spaces around them, but the `++` usually does not.

```suggestion
for (let count = 0; count <= 5; count++) {
```

Note this is just a stylistic change to conform to common standards and make your code easier for _humans_ to read. The code will run fine either way.